voxelbotutils/cogs/owner_only.py

```diff
@@ -181,7 +182,8 @@ async def downloadcog(self, ctx: utils.Context, url: str):
         except commands.NoEntryPointError:
             return await ctx.send("No added setup function.")
         except commands.ExtensionFailed:
-            return await ctx.send("Extension failed for some unknown reason.")
+            os.remove(file_path)
+            return await ctx.send("Extension failed for some unknown reason. Extension has been deleted.")
```

Comment:
There should be a comma there, not a period.

Reply:
I don't think that's necessary.